docs/HowToUsePeripheral.md

```diff
@@ -204,14 +218,14 @@ struct uart_dev_s {
 typedef struct uart_dev_s uart_dev_t;
 ```
 
-The **xmit**, **recv**, **ops** and **priv** should be defined.  
-> **xmit** : size and address of transmit buffer  
-> **recv** : size and address of receive buffer  
-> **ops**  : tables of operations like setup, send, receive and etc  
+The **xmit**, **recv**, **ops** and **priv** should be defined.
+> **xmit** : size and address of transmit buffer
+> **recv** : size and address of receive buffer
+> **ops**  : tables of operations like setup, send, receive and etc
```

Comment:
You should not remove two spaces which are at end of lines. It is a syntax of markdown.

Reply:
add two spaces.